src/metatensor/models/experimental/alchemical_model/tests/test_exported.py

```diff
@@ -35,18 +42,18 @@ def test_to(tmp_path, device, dtype):
             )
         },
         supported_devices=["cpu", "cuda"],
+        interaction_range=DEFAULT_HYPERS["model"]["soap"]["cutoff"],
+        dtype=dtype_string,
     )
-    model = Model(capabilities, DEFAULT_HYPERS["model"])
+    model = Model(capabilities, DEFAULT_HYPERS["model"]).to(dtype=dtype)
     export(model, "model.pt")
     exported = load("model.pt")
 
-    exported.to(device=device, dtype=dtype)
+    exported.to(device=device)
```

Comment:
Is passing a change in `dtype` and `device` not supported anymore?

Reply:
It might work, but now the dtype is supposed to be static in an exported model, so it's not intended use

Reply:
Okay, but then we should remove this option upstream, right?

Reply:
No, because the model will be exported either in float32 or float64 (it's higher up in the same text), but not changed after exporting